tests/test_accessibility.py

```diff
@@ -25,9 +25,68 @@ def test_accessibility_unknown():
         {
             "properties": {
                 'wheelchair': 'toto',
-                'tactile_paving': 'unknown',
             }
         },
         lang='en'
     )
     assert web_block is None
+
+def test_undefined_wheelchairs(patisserie_peron):
+    """
+    Test that when wheelchair and toilets_wheelchair are not
+    defined there is no block 'accessibility'
+    """
+    client = TestClient(app)
+    response = client.get(
+        url=f'http://localhost/v1/pois/{patisserie_peron}?lang=fr',
+    )
+
+    assert response.status_code == 200
+    assert response.headers.get('Access-Control-Allow-Origin') == '*'
+
+    resp = response.json()
+
+    assert resp['id'] == 'osm:node:738042332'
+    assert resp['name'] == "Boulangerie Patisserie Peron"
+    assert resp['local_name'] == "Boulangerie Patisserie Peron"
+    assert resp['class_name'] == 'bakery'
+    assert resp['subclass_name'] == 'bakery'
+    assert resp['blocks'] == []
+
+def test_wheelchair(cinema_multiplexe):
+    """
+    Test that when wheelchair is defined but not toilets_wheelchair
+    we still have a correct block
```

Comment:
this description does not match the test, where both values are "yes"

Reply:
Indeed: it has been fixed